perf/benchmark/configs/istio/plaintext/postrun.sh

```diff
@@ -15,8 +15,9 @@
 # limitations under the License.
 
 echo "Delete the plaintext related config..."
-kubectl delete policy -n"${NAMESPACE}" default
+kubectl delete peerauthentication.security.istio.io -n"${NAMESPACE}" default
 kubectl delete DestinationRule -n"${NAMESPACE}" plaintext-dr-twopods
 echo "Restoring original Authn Policy and DestinationRule config..."
-kubectl apply -f "${LOCAL_OUTPUT_DIR}/authn-policy.yaml"
-kubectl apply -f "${LOCAL_OUTPUT_DIR}/destionation-rule.yaml"
\ No newline at end of file
+
+[[ ${LOCAL_OUTPUT_DIR}/authn-policy.yaml ]] && kubectl apply -f "${LOCAL_OUTPUT_DIR}/authn-policy.yaml"
```

Comment:
any reason to check existence of the file? when they would not exist? Or it's just a sanity check convention?

Reply:
If there is no authn policy, those files would be empty then apply would raise error.